addons/project_hr_expense/__manifest__.py

```diff
@@ -11,6 +11,7 @@
     'depends': ['project_account', 'hr_expense'],
     'demo': [
         'data/project_hr_expense_demo.xml',
+        'views/project_project_views.xml',
```

Comment:
shouldn't that be under the data header and not the demo header ? :thinking:

Reply:
Indeed, I wonder how it could even work like that :sweat_smile: